mock-api/msw/handlers.ts

```diff
@@ -567,7 +567,7 @@ export const handlers = makeHandlers({
     // https://github.com/oxidecomputer/omicron/blob/d52aad0/nexus/db-queries/src/db/datastore/external_ip.rs#L782-L794
     // https://github.com/oxidecomputer/omicron/blob/d52aad0/nexus/src/app/sagas/instance_ip_detach.rs#L79-L82
     const ip = db.ephemeralIps.find((eip) => eip.instance_id === instance.id)
-    if (!ip) throw notFoundErr('ephemeral IP')
+    if (!ip) throw notFoundErr('Could not find an ephemeral IP')
```

Comment:
You had it right the first time. See this line:

https://github.com/oxidecomputer/console/blob/c0f9f5e78f8bf73cfce6306eb73b127129230d42/mock-api/msw/db.ts#L25

Reply:
Ah, womp; ty